routes/clarivate.js

```diff
@@ -208,33 +189,63 @@ function getAdmin(req, res, uri, callback) {
         callback);
 };
 
-function getFilteredConsumerId(req,res) {
-  getAdmin(req, res, '/consumers/'+req.query.consumerid, (err, consumer) => {
-      if (err) {
-          return next(err);
-      }
-  let body = utils.getJson(consumer.body);
-      if (body && body.message === "Not found") {
-          res.json({
-              title: 'All Subsriptions',
-              subscriptions: null
-          });
-      } else {
-          //Desctruct body.username(for example app1$mockbin) and assign to variables
-          let [appId, apiId] = (body.username).split("$");
-          utils.getFromAsync(req, res, `/applications/${appId}/subscriptions/${apiId}`, 200, (err, application) => {
-              if (err) {
-                  return next(err);
-              }
-              application.consumerid = req.query.consumerid;
-      let subscriptions = {items: [application],count: 1}
-              res.json({
-                  title: 'All Subsriptions',
-                  subscriptions
-              });
-          })
-      }
-  });
+function getSubscriptions(req, res,) {
```

Comment:
no need for coma at the end of the arguments?

Reply:
Thank you. Fixed